src/CreditCardInput.js

```diff
@@ -86,12 +87,15 @@ export default class CreditCardInput extends Component {
     invalidColor: "red",
     placeholderColor: "gray",
     allowScroll: false,
+    disableAutoFocus: false,
   };
 
-  componentDidMount = () => this._focus(this.props.focused);
+  componentDidMount = () => {
+    if (!this.props.disableAutoFocus) this._focus(this.props.focused);
+  }
 
   componentWillReceiveProps = newProps => {
-    if (this.props.focused !== newProps.focused) this._focus(newProps.focused);
```

Comment:
Wouldn't this make the `focused` prop & what actually focused out of sync? 
e.g. given `focused="number"`, other text input might be on focus instead